RecoLocalTracker/SiStripClusterizer/python/SiStripClusterizer_RealData_cfi.py

```diff
@@ -21,3 +21,6 @@
                                      cms.InputTag('siStripZeroSuppression','ScopeMode'))
 )
 
+# the input digis from ZS (on top of hybrid format) should be changed -- for HI tests
+from Configuration.Eras.Era_Run2_HI_2018_cff import run2_HI_2018
```

Comment:
please import from Modifier_run2_HI_2018_cff.
this is more direct and will keep working even if the content of Run2_HI_2018 chain changes.